src/scripts/components/main/navigation/navigation.tsx

```diff
@@ -65,6 +81,7 @@ const Navigation: FunctionComponent = () => {
           <Menu />
         </Overlay>
       )}
+      {isFilterActive && showTags && <TagsBubble selectedTags={selectedTags} />}
```

Comment:
The tag selector is not visible when no tag is selected? How can you select a tag then? Also TagsBubble is not a very specific name, I wasn't sure what this component does when reading it.

Reply:
The tags are selected in the stories, the user is not supposed to do that in the data view: https://github.com/ubilabs/esa-climate-from-space/issues/823